### PR TITLE
BUG: Fix bug with median bias

### LINKED ISSUE #10959: mne.time_frequency.psd_welch with option average=median
In the function `mne.time_frequency.psd_welch`, for the option `average` it is written:
_How to average the segments. If mean (default), calculate the arithmetic mean. If median, calculate the median, corrected for its bias relative to the mean. If None, returns the unaggregated segments._

I searched through the code, and am not finding any correction of the median for the bias relative to mean.
1. Is it actually done somewhere and I missed?
2. Would you have some references about how such bias correction is done?

### PR DESCRIPTION
**Pull request by larsoner**
Closes #10959

### CODE REVIEW

**Comment by larsoner** [comment]
After this I think we can start the 1.1 release process!

**Comment by larsoner** [comment]
@drammock this one should be good to go

**Review by drammock — APPROVED** [review]

File: mne/time_frequency/psd.py [hunk 1]
```diff
@@ -11,14 +11,21 @@
...
+
 def _decomp_aggregate_mask(epoch, func, average, freq_sl):
     _, _, spect = func(epoch)
     spect = spect[..., freq_sl, :]
     # Do the averaging here (per epoch) to save memory
     if average == 'mean':
         spect = np.nanmean(spect, axis=-1)
     elif average == 'median':
-        spect = np.nanmedian(spect, axis=-1)
+        spect = np.nanmedian(spect, axis=-1) / _median_bias(spect.shape[-1])
```
Comment (chapochn) [inline mne/time_frequency/psd.py:28]: I don't think the bias calculation is correct in the case where there are nans in the spect. I believe _median_bias needs to take the number of non nans in its argument, no?

Comment (mmagnuski) [reply mne/time_frequency/psd.py:28]: I agree, the n should be something like `np.sum(~np.isnan(data))`.

**Pull request merged**